### PR TITLE
CoC: Add first commit

### LINKED ISSUE #297: adding CONTRIBUTING.md and COC.md file
I will like to take this issue

### PR DESCRIPTION
**Pull request by patrickelectric**
Fix #297

### CODE REVIEW

**Review by joaoantoniocardoso — APPROVED** [review]

**Comment by rafaellehmkuhl** [comment]
A 5a serie...

File: CoC.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+Examples of unacceptable behavior by participants include:
+
+- The use of sexualized language or imagery and unwelcome sexual attention or advances
+- Trolling, insulting/derogatory comments, and personal or political attacks
+- Public or private harassment
+- Publishing others’ private information, such as a physical or electronic address, without explicit permission
+- Other conduct which could reasonably be considered inappropriate in a professional setting
+
+## Our Responsibilities
+BlueOS, maintainers, project participants, and contributors (collectively, “participants”) are responsible for clarifying the standards of acceptable behavior and are expected to take appropriate and fair corrective action in response to any instances of unacceptable behavior.
```
Comment (ES-Alexander) [inline CoC.md:30]:
```suggestion
BlueOS maintainers, project participants, and contributors (collectively, “participants”) are responsible for clarifying the standards of acceptable behavior and are expected to take appropriate and fair corrective action in response to any instances of unacceptable behavior.
```

File: CoC.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+Blue Robotics has the right and responsibility to remove, edit, or reject comments, commits, code, wiki edits, issues, and other contributions that are not aligned to this Code of Conduct, or to ban temporarily or permanently any contributor for other behaviors that they deem inappropriate, threatening, offensive, or harmful.
```
Comment (ES-Alexander) [inline CoC.md:32]:
```suggestion
As the primary project maintainer, Blue Robotics has the right and responsibility to remove, edit, or reject comments, commits, code, wiki edits, issues, and other contributions that are not aligned to this Code of Conduct, or to ban temporarily or permanently any contributor for other behaviors that they deem inappropriate, threatening, offensive, or harmful.
```
Seems worth contextualising _why_ we have those rights and responsibilities :-)

File: CoC.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+# Reporting and Enforcement
+Report unacceptable behavior to the project maintainers (software@bluerobotics.com). All complaints will be reviewed and investigated promptly and fairly. Consequences for violations may include temporary or permanent bans from the project.
+
+# Conclusion
+### By participating in this project, you agree to uphold this Code of Conduct. Let's work together to create a community where everyone feels safe and welcome, with self-love, respect, inclusivity, positivity, and accountability.
+
+
+---
+> Ref: This CoC is was originaly based on [Dronecode Foundation CoC](https://discuss.px4.io/t/px4-community-code-of-conduct/13655)
```
Comment (ES-Alexander) [inline CoC.md:45]:
```suggestion
> Ref: This CoC is adapted from the [Dronecode Foundation CoC](https://discuss.px4.io/t/px4-community-code-of-conduct/13655)
```

**Review by ES-Alexander — CHANGES_REQUESTED** [review]
Mostly looks good. A few minor suggestions :-)

**Comment by patrickelectric** [comment]
@ES-Alexander check now

**Review by ES-Alexander — APPROVED** [review]
Love it! :-)

**Comment by ES-Alexander** [comment]
Docs should mention this in the development overview :-)

**Pull request merged**